lib/hackmode/realmode.js

```diff
@@ -171,8 +171,8 @@ class N3hRealMode extends N3hMode {
     log.i(this.me() + 'p2p bound', this._p2p.getAdvertise())
   }
 
-  async _sendToAgent (msg, dnaAddress, toAgentId, fromAgentId, reqId) {
-    const mId = this._getPeerAddressOrFail(dnaAddress, toAgentId, fromAgentId, reqId)
+  async _sendToAgent (msg, dnaAddress, agentId, toAgentId, fromAgentId, reqId) {
```

Comment:
If the `agentId` here is referring to the agentId associated with the chainId, might this be called `chainAgentId` for clarity?